src/containers/rendering/Variables.jsx

```diff
@@ -43,10 +48,23 @@ class MyFuckingVariables extends React.Component {
             <ButtonGroup
               color="primary"
               aria-label="outlined primary button group"
+              classNames={classes.arrays}
             >
-              {_.get(variable, "values", []).map((index, i) => (
-                <Button key={variable.name + "-" + i}>{index}</Button>
-              ))}
+              {_.get(variable, "values", []).map((index, i) => {
+                if (_.get(variable, "values", []).length === pointers.length && (pointers[i].length > 0)) {
+                  let pointer_names = "";
+                  pointers[i].forEach(pointer_name => {
+                    pointer_names += pointer_name + " "
```

Comment:
maybe add comma and space, if you do this then you need to not include the last char of the of `pointer_names`